mongodbatlas/version.go

```diff
@@ -0,0 +1,15 @@
+package mongodbatlas
+
+import "fmt"
+
+const (
+	// UserAgent the default user agent string reported by this client
+	UserAgent = "go-mongodbatlas"
+	// ClientVersion the version of the current API client
+	ClientVersion = "0.4.0" // Should be set to the next version planned to be released
```

Comment:
it's more common to just say `Version`
```suggestion
	// Version the version of the current API client
	Version = "0.4.0" // Should be set to the next version planned to be released
```

Reply:
version is very generic though; hard to search for

Reply:
still, updated since it's more go idiomatic